qa/runtest.sh

```diff
@@ -20,12 +20,12 @@ check_python_version
 if [[ "$CI" -eq "0" ]];
 then
     make docker-build-test
-    docker run ${TEST_IMAGE_NAME}
+    docker run  -v "$PWD:/shared:rw,Z" ${TEST_IMAGE_NAME}
```

Comment:
Don't use /shared and separate image for testing.

I guess this doesn't even need docker to run tests., it shall be removed.